letsencrypt/client/configuration.py

```diff
@@ -48,6 +48,18 @@ def cert_key_backup(self):  # pylint: disable=missing-docstring
             self.namespace.work_dir, constants.CERT_KEY_BACKUP_DIR,
             self.namespace.server.partition(":")[0])
 
+    @property
+    def accounts_dir(self):  #pylint: disable=missing-docstring
+        return os.path.join(
+            self.namespace.config_dir, constants.ACCOUNTS_DIR,
+            self.namespace.server.partition(":")[0])
```

Comment:
No, I meant confusing for the user as an end-entity.  I was concerned with confusing the user with multiple accounts for technically the same ACME instance.  Accounts now store recovery_tokens and recovery_contacts (maybe existing authorizations in the future assuming they have some timeout).  If the "server" changes, a new account will be created which will yield a new recovery_token and a recovery_token challenge when the user goes to authorize.

I do agree with you though to fully specify the server path as it is a fatal bug if we do run into the multiple instance scenario.